### PR TITLE
Add generator for simplecov

### PR DESCRIPTION
**Pull request by avdgaag**

### CODE REVIEW

File: styleguide [hunk 1]
```diff
@@ -244,12 +244,63 @@ module Styleguide
...
+    def inject_into_test_helper
+      filename = 'spec/spec_helper.rb'
+      filename = 'test/test_helper.rb' unless File.exist?(filename)
+      return unless File.exist?(filename)
+      return if file_contains? filename, 'SimpleCov'
+      if options[:defaults]
+        prepend_to_file filename, "# Configure SimpleCov in <root>/.simplecov\nrequire 'simplecov' unless ENV['SKIP_COVERAGE']\n"
+      else
+        prepend_to_file filename, <<-EOS
+unless ENV['DISABLE_COVERAGE']
```
Comment (florish) [inline styleguide]: Moet dit niet zijn `SKIP_COVERAGE`?

File: styleguide [hunk 2]
```diff
@@ -244,12 +244,63 @@ module Styleguide
     end
   end
 
+  class Simplecov < Thor::Group
+    include Helpers
+
+    class_option :defaults, default: true, type: :boolean, desc: 'Whether to import the default configuration'
```
Comment (florish) [inline styleguide]: I'm not sure what this option means, I just left it to the default (`true`) but why wouldn't I want to import the defaults?

**Comment by florish** [comment]
@avdgaag Looks fine to me in general, see line comments for some questions.

More of a conceptual question, though: is the styleguide repository the right place for `simplecov`? Same question for `bundler-audit`, which was already in here, but also seems a bit off in terms of what it does.

If you agree, there are two options I can think of right now:

* Move `simplecov` and `bundler-audit` to a separate repository
* Rename `styleguide` to clarify it's about coding style, security, basically code quality in general.

Both have pros and cons, and there are probably other options, too 😄

File: styleguide (hunk 1 above)
Comment (avdgaag) [inline styleguide]: Yes it shoukd.

File: styleguide (hunk 2 above)
Comment (avdgaag) [inline styleguide]: Because you have an old config? You don't like them? Want something simple? All generators have this option.

**Comment by avdgaag** [comment]
I will probably rename this repository.

File: styleguide (hunk 2 above)
Comment (florish) [inline styleguide]: Ok, sounds reasonable :)

**Comment by florish** [comment]
@avdgaag OK. Just so we don't forget: we should probably add `/coverage` to `.gitignore` to avoid coverage reports from being checked into repositories.

**Comment by avdgaag** [comment]
> we should probably add /coverage to .gitignore to avoid coverage reports from being checked into repositories.

Done, merging now.

**Pull request merged**